pom.xml

```diff
@@ -234,7 +234,11 @@
         </exclusion>
       </exclusions>
     </dependency>
-
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>snakeyaml-api</artifactId>
+      <version>2.0-999999-SNAPSHOT</version>    
```

Comment:
An incremental version was deployed https://ci.jenkins.io/job/Plugins/job/snakeyaml-api-plugin/job/PR-94/1/pipeline-console/ but how to figure out what that version is I don't know.

Reply:
Found it!
```suggestion
      <version>2.1-110.v4960f4074978</version>    
```